nsot/api/views.py

```diff
@@ -287,12 +287,32 @@ def perform_destroy(self, instance):
         change = models.Change.objects.create(
             obj=instance, user=self.request.user, event='Delete'
         )
+        force_delete = qpbool(
+            self.request.query_params.get('force_delete', False)
+        )
 
         try:
             instance.delete()
         except exc.ProtectedError as err:
-            change.delete()
-            raise exc.Conflict(err.args[0])
+            if force_delete:
```

Comment:
Strange that these changes are making it in just now. It looks like it's from #337, which didn't find its way into the master branch nor the 1.4.2 release (based on the tag):
https://github.com/dropbox/nsot/commits/v1.4.2

This is fine since it's already in the changelog anyway. You may want to consider updating the changelog to move the bit about the forceful-delete to be under 1.4.3 rather than 1.4.2.